components/AuthorBox.tsx

```diff
@@ -28,9 +47,16 @@ const AuthorBox: React.ElementType<AuthorBoxProps> = ({
   axios
     .get("https://wp.stanforddaily.com/wp-json/tsd/v1/authors/" + id + "/")
     .then(response => {
+      console.log(response);
       setAdditionalAuthorInfo(response.data);
     });
 
+  // set object returns to local variables
+  const emailURL = "mailto:" + encodeURIComponent(additionalAuthorInfo?.email);
```

Comment:
Can we change all instances of `.email` to `.dailyEmail`? Made this change on the backend